<define-tag description>security update</define-tag>
<define-tag moreinfo>
<p>Several vulnerabilities were discovered in Django, a high-level Python
web development framework. The Common Vulnerabilities and Exposures
project identifies the following problems:</p>

<ul>

<li><a href="https://security-tracker.debian.org/tracker/CVE-2016-9013">CVE-2016-9013</a>

    <p>Marti Raudsepp reported that a user with a hardcoded password is
    created when running tests with an Oracle database.</p></li>

<li><a href="https://security-tracker.debian.org/tracker/CVE-2016-9014">CVE-2016-9014</a>

    <p>Aymeric Augustin discovered that Django does not properly validate
    the Host header against settings.ALLOWED_HOSTS when the debug
    setting is enabled. A remote attacker can take advantage of this
    flaw to perform DNS rebinding attacks.</p></li>

<li><a href="https://security-tracker.debian.org/tracker/CVE-2017-7233">CVE-2017-7233</a>

    <p>It was discovered that is_safe_url() does not properly handle
    certain numeric URLs as safe. A remote attacker can take advantage
    of this flaw to perform XSS attacks or to use a Django server as an
    open redirect.</p></li>

<li><a href="https://security-tracker.debian.org/tracker/CVE-2017-7234">CVE-2017-7234</a>

    <p>Phithon from Chaitin Tech discovered an open redirect vulnerability
    in the django.views.static.serve() view. Note that this view is not
    intended for production use.</p></li>

</ul>

<p>For the stable distribution (jessie), these problems have been fixed in
version 1.7.11-1+deb8u2.</p>

<p>We recommend that you upgrade your python-django packages.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2017/dsa-3835.data"
# $Id$
